### PR TITLE
FIX: Removed remaining strings from removed package

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/i37wiv6kybtwicvjepjqxawusq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12229)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: components/WalletsCarousel.js [hunk 1]
```diff
@@ -176,10 +176,10 @@ const WalletCarouselItem = ({ item, _, onPress, handleLongPress, isSelectedWalle
...
       ? loc.transactions.updating
       : item.getBalance() !== 0 && item.getLatestTransactionTime() === 0
-      ? loc.wallets.pull_to_refresh
-      : item.getTransactions().find(tx => tx.confirmations === 0)
-      ? loc.transactions.pending
-      : transactionTimeToReadable(item.getLatestTransactionTime());
+        ? loc.wallets.pull_to_refresh
+        : item.getTransactions().find(tx => tx.confirmations === 0)
+          ? loc.transactions.pending
+          : transactionTimeToReadable(item.getLatestTransactionTime());
```
Comment (Overtorment) [inline components/WalletsCarousel.js:182]: this shit is unreadable tbh

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**